src/NuGet.Core/NuGet.Protocol.Core.v3/Model/PackageSearchMetadata.cs

```diff
@@ -96,5 +96,7 @@ public string Title
         public VersionInfo[] ParsedVersions { get; private set; }
 
         public Task<IEnumerable<VersionInfo>> GetVersionsAsync() => Task.FromResult<IEnumerable<VersionInfo>>(ParsedVersions);
+        //The V3 source currently does return a listed property
```

Comment:
If so, why not plumb the value through?

---
In reply to: [94631214](https://github.com/NuGet/NuGet.Client/pull/1088#discussion_r94631214) [](ancestors = 94631214)